tests/e2e/package.json

```diff
@@ -37,10 +37,10 @@
     "@types/node": "11.13.4",
     "@types/rimraf": "2.0.2",
     "@types/selenium-webdriver": "3.0.16",
-    "axios": "0.21.2",
```

Comment:
There is no information about an update of axios in PR description.

What was the reason to do change the version from 0.21.2 to 0.24.0?

Reply:
Its updated as part of npm audit fix, but it doesn't have any major feature changes. On re-running audit-fix command, this package version is not getting updated. Let me revert this change. Maybe later we can think about upgrading the packages.

Reply:
> @makambalaji: looks like [recent changes](https://github.com/eclipse/che/pull/20892/commits/6e1289997a8961f7a4ac3c637401c9125dbf8beb) have broken Happy path tests https://main-jenkins-csb-crwqe.apps.ocp4.prod.psi.redhat.com/job/Che/job/e2e/job/minikube/job/pr-check/job/che-tests-pr-check/298/console :
> 
> ```
>  ​> @eclipse-che/che-e2e@7.41.0-SNAPSHOT sc /tmp/e2e
> ​>tsc -p .
>  ​
>  ​utils/requestHandlers/CheApiRequestHandler.ts(28,21): error TS2532: Object is possibly undefined'.
>  ​utils/requestHandlers/CheApiRequestHandler.ts(50,21): error TS2532: Object is possibly undefined'.
>  ​npm ERR! cod ELIFECYCLE
>  ​npm RR! errno 2
>  ​npm ERR! @eclipse-che/che-e2e@7.41.0-SNAPSHOT tsc `tsc -p .`
>  ​npm ERR! Eit status 2 ​npm ERR! 
>  ​npm ERR! Failed at the @eclipse-che/che-e2e@7.41.0-SNAPSHOTtsc script.
>  ​npm ERR! This is probably not a problem with npm. There is likely additional logging otpt above.
> ```
> 
> Take a look, please.

Removed the axios version upgrade, looks like that is the issue related to this error

Reply:
Thank you!